### PR TITLE
Feat/220 feat web version

### LINKED ISSUE #261: [Bug]: Web Bugs
### Contact Details

-

### Branch name

development

### Flutter & Dart version

web

### How are you running it?

I'm running the app hosted in dev.ottaaproject.com on a MacBook with Chrome.

- [x] Increase the speed of the TTS, all available options are slow.
- [x] es_CO throws an error when choosing. 
- [x] Check how the TTS is fetching the available voices because it's very weird and not following the language.
- [x] #266
- [x] Fix the error when the picto does not have relations with other pictos. 

### What happened?

A bug happened!

### Where are you seeing the problem on?

Chrome (WebApp)

### Relevant log output

_No response_

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

### LINKED ISSUE #264: [Bug]: Bugs after merging #262
### Contact Details

_No response_

### Branch name

development with #262

### Flutter & Dart version

Yes

### How are you running it?

Bugs before merging:
User:
- [x] Click on "Tableros/Pictogramas" and when pressing the device "Back button" shows a dialog "Esta characteristics va a star disponible muy pronto"

My steps are flutter clean...pub get... re do the models and run. 

Bugs after merging:
User

- [x] Error 1: Unhandled exception No element 
- [x] Pictogram when loading is bigger than the Picto with the information compare Picture 1 and Picture 3 
- [x] Pictogram "Add" is bigger than the other one. Picture 3
- [x] Go to Boards/Pictograms and then click "Guardar Cambios" throws Error 2: if you click again throws Error 3
- [x] Even though #221 is merged the bug still appears.  Picture 4

Caregiver
- [x] When linking a new user, you get an error after the linking is correct. Does not open the Boards/Pictograms screen. 
- [x] When linking a new user, the notification does not arrive on all devices. 
- [x] There is an error loading the Boards from the Caregiver side. It doesn't happen everytime and does not throws any log or error in the console. Video 1

Error 1:
[ERROR:flutter/runtime/dart_vm_initializer.cc(41)] Unhandled Exception: Bad state: No element E/flutter (18032): #0      ListMixin.firstWhere (dart:collection/list.dart:167:5) E/flutter (18032): #1      new LayoutSetting.fromMap (package:ottaa_project_flutter/core/models/layout_setting.dart:64:12) E/flutter (18032): #2      new PatientSettings.fromMap (package:ottaa_project_flutter/core/models/patient_user_model.dart:240:53) E/flutter (18032): #3      new PatientUserModel.fromMap (package:ottaa_project_flutter/core/models/patient_user_model.dart:158:33) E/flutter (18032): #4      AboutService.getUserInformation (package:ottaa_project_flutter/application/service/about_service.dart:163:34) E/flutter (18032): <asynchronous suspension> E/flutter (18032): #5      SplashProvider.fetchUserInformation (package:ottaa_project_flutter/application/providers/splash_provider.dart:36:20) E/flutter (18032): <asynchronous suspension> E/flutter (18032): #6      _SplashScreenState.initState.<anonymous closure> (package:ottaa_project_flutter/presentation/screens/splash/splash_screen.dart:41:23)
E/flutter (18032): <asynchronous suspension>

Error 2:
E/flutter (18653): [ERROR:flutter/runtime/dart_vm_initializer.cc(41)] Unhandled Exception: A CustomiseProvider was used after being disposed.
E/flutter (18653): Once you have called dispose() on a CustomiseProvider, it can no longer be used.
E/flutter (18653): #0      ChangeNotifier.debugAssertNotDisposed.<anonymous closure> (package:flutter/src/foundation/change_notifier.dart:157:9)
E/flutter (18653): #1      ChangeNotifier.debugAssertNotDisposed (package:flutter/src/foundation/change_notifier.dart:164:6)
E/flutter (18653): #2      ChangeNotifier.notifyListeners (package:flutter/src/foundation/change_notifier.dart:360:27)
E/flutter (18653): #3      CustomiseProvider.notify (package:ottaa_project_flutter/application/providers/customise_provider.dart:144:5)
E/flutter (18653): #4      _CustomizedMainTabScreenState.build.<anonymous closure> (package:ottaa_project_flutter/presentation/screens/customized_board/customized_board_tab_screen.dart:157:38)
E/flutter (18653): <asynchronous suspension>

Error 3:
The following _CastError was thrown building CustomizeWaitScreen(dirty, dependencies: [_InheritedTheme, _LocalizationsScope-[GlobalKey#30c33]], state: _ConsumerState#7a2bf):
Null check operator used on a null value

The relevant error-causing widget was: 
  CustomizeWaitScreen CustomizeWaitScreen:file:///C:/Users/Agus/StudioProjects/ottaa_project_flutter/lib/application/router/app_router.dart:184:56
When the exception was thrown, this was the stack: 
#0      CustomizeWaitScreen.build (package:ottaa_project_flutter/presentation/screens/customized_board/customized_wait_screen.dart:52:53)
#1      _ConsumerState.build (package:flutter_riverpod/src/consumer.dart:477:19)
#2      StatefulElement.build (package:flutter/src/widgets/framework.dart:5080:27)
#3      ConsumerStatefulElement.build (package:flutter_riverpod/src/consumer.dart:538:20)
#4      ComponentElement.performRebuild (package:flutter/src/widgets/framework.dart:4968:15)
#5      StatefulElement.performRebuild (package:flutter/src/widgets/framework.dart:5133:11)
#6      Element.rebuild (package:flutter/src/widgets/framework.dart:4690:5)
#7      ComponentElement._firstBuild (package:flutter/src/widgets/framework.dart:4950:5)
#8      StatefulElement._firstBuild (package:flutter/src/widgets/framework.dart:5124:11)
#9      ComponentElement.mount (package:flutter/src/widgets/framework.dart:4944:5)

Picture 1:
image

Picture 2:
image

Picture 3:
image

Picture 4:
image

Video 1:

[error_loading_boards.webm](https://user-images.githubusercontent.com/12820318/235250660-2fbcd6f1-71b6-4a42-ae13-3e3f277cf832.webm)

### What happened?

A bug happened!

### Where are you seeing the problem on?

_No response_

### Relevant log output

_No response_

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

### PR DESCRIPTION
**Pull request by emirchus**
We added a best behaviour to some screens and fixed some errors!

Fixed: #264 
Fixed: #261

### CODE REVIEW

**Comment by emirchus** [comment]
@hectoritr all issues should be done and ready to test

File: assets/games_images/ottaa.png
Comment (hectoritr) [inline assets/games_images/ottaa.png:1]: What's this icon, and why is called OTTAA. That is not OTTAA logo.

Comment (hectoritr) [reply assets/games_images/ottaa.png:1]: @asimjawad

**Review by hectoritr — APPROVED** [review]
Looks good

**Comment by Da-PC** [comment]
@emirchus still failing with Pixel 6. I think it is not a diagonal lenght issue but a density of pixels where its failing. 

image

File: assets/games_images/ottaa.png
Comment (asimjawad) [reply assets/games_images/ottaa.png:1]: I have changed its name. I might have left it unchecked.

**Comment by emirchus** [comment]
> @emirchus still failing with Pixel 6. I think it is not a diagonal lenght issue but a density of pixels where its failing.
> 
> image

Ok, I will working with dp instead of px

**Pull request merged**